settings.gradle

```diff
@@ -1 +1,17 @@
 include ':app'
+pluginManagement.repositories {
+    maven {
+        url "https://plugins.gradle.org/m2/"
+    }
+    google()
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+    maven {
+        url 'https://artifactory.rightmesh.io/artifactory/maven'
+        credentials {
+            username rightmesh_artifactory_username
```

Comment:
Just to be clear:
I think you had a line before that mapped the `devportal_username` to this. This _can_ be the `rightmesh_devportal_username` correct? Then we'd have people define the username and the two passwords and we're good for now?

Reply:
:grin: